go/client/cmd_wot_list.go

```diff
@@ -29,28 +30,43 @@ func newCmdWotList(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comma
 		Action: func(c *cli.Context) {
 			cl.ChooseCommand(cmd, "list", c)
 		},
-		Flags: []cli.Flag{},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "by-me",
```

Comment:
chose to only expose this for now; it's easy to replace this with a StringFlag `voucher` for the voucher username.

Reply:
replaced with `voucher` string arg